.circleci/config.yml

```diff
@@ -33,9 +33,10 @@ jobs:
           name: Push Docker image
           command: |
             docker login -u $DOCKERCLOUD_USER -p $DOCKERCLOUD_PASS
-            docker tag mirumee/saleor:latest mirumee/saleor:$CIRCLE_SHA1
+            docker tag mirumee/saleor:latest mirumee/saleor:$CIRCLE_SHA1 mirumee/saleor:$SALEOR_VERSION
             docker push mirumee/saleor:$CIRCLE_SHA1
             docker push mirumee/saleor:latest
+            docker push mirumee/saleor:$SALEOR_VERSION
```

Comment:
Sounds like it would push for every build on master instead of for every release?

Reply:
On every push to master `SALEOR_VERSION` already had the value of `git rev-parse --short HEAD` but was not used except on image build. So also pushing that tag to Docker Hub. Anyway can remove that if you do think it's better.